src/index.html

```diff
@@ -9,10 +9,86 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+
+      <p
+      class="card__name"
+      >
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p
+      class="card__code"
+      >
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p
+        class="card__rating-reviews"
+        >
```

Comment:
Fix it everywhere if you have only 1 attribute
```suggestion
        <p class="card__rating-reviews">
```